tests/components/aquacell/test_init.py

```diff
@@ -0,0 +1,29 @@
+"""Test the Aquacell init module."""
+
+from __future__ import annotations
+
+from unittest.mock import AsyncMock
+
+from homeassistant.components.aquacell.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+from tests.components.aquacell import setup_integration
+
+
+async def test_load_unload_entry(
+    hass: HomeAssistant,
+    mock_aquacell_api: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test load and unload entry."""
+    await setup_integration(hass, mock_config_entry)
+    entry = hass.config_entries.async_entries(DOMAIN)[0]
```

Comment:
You can just use `mock_config_entry` here, it's the same object as you're fetching now